adm_program/system/classes/htmlform.php

```diff
@@ -269,32 +267,17 @@ public function addCaptcha($id, $type, $class = '')
         // add a row with the captcha puzzle
         $this->openControlStructure('captcha_puzzle', '');
 
-        $captchaLabel = '';
-        $captchaDescription = '';
-
-        if($type === 'pic')
-        {
-            $this->addHtml('<img src="'.$g_root_path.'/adm_program/system/show_captcha.php?id='.time().'" alt="'.$gL10n->get('SYS_CAPTCHA').'" />');
-            $captchaLabel = $gL10n->get('SYS_CAPTCHA_CONFIRMATION_CODE');
-            $captchaDescription = 'SYS_CAPTCHA_DESCRIPTION';
-        }
-        elseif($type === 'calc')
-        {
-            $captcha = new Captcha();
-            $this->addHtml($captcha->getCaptchaCalc($gL10n->get('SYS_CAPTCHA_CALC_PART1'),
-                                                    $gL10n->get('SYS_CAPTCHA_CALC_PART2'),
-                                                    $gL10n->get('SYS_CAPTCHA_CALC_PART3_THIRD'),
-                                                    $gL10n->get('SYS_CAPTCHA_CALC_PART3_HALF'),
-                                                    $gL10n->get('SYS_CAPTCHA_CALC_PART4')));
-            $captchaLabel = $gL10n->get('SYS_CAPTCHA_CALC');
-            $captchaDescription = 'SYS_CAPTCHA_CALC_DESCRIPTION';
-        }
+        $captchaLabel = $gL10n->get('SYS_CAPTCHA_CONFIRMATION_CODE');
+        $captchaDescription = 'SYS_CAPTCHA_DESCRIPTION';
```

Comment:
why you keep this 2 variables ($captchaLabel and $captchaDescription) in the code?
i can not see any useage of them?